gensim/utils.py

```diff
@@ -1146,15 +1146,16 @@ def keep_vocab_item(word, count, min_count, trim_rule=None):
         else:
             return default_res
 
-def check_output(*popenargs, **kwargs):
+def check_output(*popenargs, stdout=subprocess.PIPE, **kwargs):
```

Comment:
this will keep the previous default `stdout=subprocess.PIPE` for other wrappers, and a different `stdout` can be defined for wordrank wrapper